### ISSUE TITLE
Continuous Integration Tutorial

### ISSUE THREAD

**Issue by MichaMans** [RWTH-EBC/EBC-Tutorials#20]
I would write a few sentences about the integration of CI in your software projects. Therefore i would like to invite everybody to discuss where to add this. I see the following possibilities:
- Git-Tutorial at "What's next" section
- Python Tutorial with own new section (maybe with an additional pytest howto link or something)
- AixLib or TEASER Wiki
- gitlab wiki, whereas i would integrate a gitlab runner specific section at the show above too

Feel free to share your thoughts!

**Comment by JSchiefelbein** [comment]
Good idea (and important question). As Continuous Integration is relevent for different programming languages, it might be reasonable to open an own chapter about CI. This could be linked/referenced from specific tutorials/wikis (Python Tutorial, AixLib etc.). 
Moreover, we could add some pytest examples to the Python Tutorial section. 

Just my 5 cents....

**Comment by marcusfuchs** [comment]
I agree, after choosing some place, linking there from everywhere seems to be a good approach. A top-level tutorial in EBC tutorials may be a good place for this.

**Comment by MichaMans** [comment]
I think I'll create a branch in the next time introducing a short part in the python tutorial for how to write tests and link to travis and gitlab tutorials for CI.

- [x] Python "How to write tests" section
- [ ] Links to Travis and gitlab

**Comment by PMehrfeld** [comment]
@MichaMans : I like the idea of this issue. Any updates?
